dev_setup.sh

```diff
@@ -301,13 +314,10 @@ function install_deps() {
             cd ..
             rm -rf fann
         )
-    elif found_exe dnf ; then
-        # Fedora
-        $SUDO dnf install -y git python3 python3-devel python3-pip python3-setuptools python3-virtualenv pygobject3-devel libtool libffi-devel openssl-devel autoconf bison swig glib2-devel portaudio-devel mpg123 mpg123-plugins-pulseaudio screen curl pkgconfig libicu-devel automake libjpeg-turbo-devel fann-devel gcc-c++ redhat-rpm-config jq
     else
     	echo
         echo "${GREEN}Could not find package manager"
-        echo "${GREEN}Make sure to manually install:${BLUE} git python 2 python-setuptools python-virtualenv pygobject virtualenvwrapper libtool libffi openssl autoconf bison swig glib2.0 portaudio19 mpg123 flac curl fann g++"
+        echo "${GREEN}Make sure to manually install:${BLUE} git python3 python-setuptools python-venv pygobject libtool libffi libjpg openssl autoconf bison swig glib2.0 portaudio19 mpg123 flac curl fann g++ jq"
```

Comment:
Green seems like an odd color to use here.  I usually consider green to be an "OK" and nothing needs to be done.  Yellow means you might need to do something as a warning.

How about making this message YELLOW instead of GREEN.  Then change the above messages that show which either BLUE or GREEN? (as noted above)